logger/mask.go

```diff
@@ -179,7 +181,14 @@ func getMaskedValue(maskChar, valueToReplace string, toMask MaskData) string {
 		return valueToReplace
 	}
 
-	totalCharsToShow := toMask.FirstCharsToShow + toMask.LastCharsToShow
+	firstCharsToShow := toMask.FirstCharsToShow
+	for _, prefix := range toMask.prefixesToSkip {
+		if strings.HasPrefix(valueToReplace, prefix) {
+			firstCharsToShow = len(prefix) + toMask.FirstCharsToShow
```

Comment:
should we put a `break` here once we find the prefix?

Reply:
Done in 93c1c41.